src/ImageSharp/MetaData/Profiles/ICC/Curves/IccFormulaCurveElement.cs

```diff
@@ -0,0 +1,84 @@
+﻿namespace ImageSharp
+{
+    /// <summary>
+    /// A formula based curve segment
+    /// </summary>
+    internal sealed class IccFormulaCurveElement : IccCurveSegment
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="IccFormulaCurveElement"/> class.
+        /// </summary>
+        /// <param name="type">The type of this segment (0-2)</param>
+        /// <param name="gamma">Gamma segment parameter</param>
+        /// <param name="a">A segment parameter</param>
+        /// <param name="b">B segment parameter</param>
+        /// <param name="c">C segment parameter</param>
+        /// <param name="d">D segment parameter</param>
+        /// <param name="e">E segment parameter</param>
+        public IccFormulaCurveElement(ushort type, double gamma, double a, double b, double c, double d, double e)
+            : base(IccCurveSegmentSignature.FormulaCurve)
+        {
+            Guard.MustBeBetweenOrEqualTo(type, 0, 2, nameof(type));
+
+            this.Type = type;
+            this.Gamma = gamma;
+            this.A = a;
+            this.B = b;
+            this.C = c;
+            this.D = d;
+            this.E = e;
+        }
+
+        /// <summary>
+        /// Gets the type of this curve
+        /// </summary>
+        public ushort Type { get; }
```

Comment:
It should be yes, I remember wanting to do this but didn't know how to name it. Implemented it now with member names Type1-3. Maybe I should come back to this and give them some better names.
(IccParametricCurve had the same problem, did that too now)